### DISCUSSION TITLE
pages directory not created in new Next.js project

### DISCUSSION THREAD

**Discussion by tacmass** [vercel/next.js#58278] [Help]
### Summary

Hello, I've created a new project using the latest version of Next.js (14.0.1), but the pages directory is not being generated in my project folder.
npx create-next-app@latest
There were no errors during the process, and other files and node_modules were created successfully. Can anyone suggest what might be causing this issue and how I can ensure the pages directory is created?

**Comment by icyJoseph** [comment]
You might have selected `App Router` when answering the questions?

**Reply by prashantrao23** [reply]
But isn't selecting App router is a recommended option?

**Comment by Harsh-Aakoliya** [comment]
while installing next app using 
`npx create-next-app@latest`
and while answering below question
if you have choosen "YES" for
1) Would you like to use `src/` directory?
2) Would you like to use App Router? (recommended)
then you might not found that pages folder.
![image](https://github.com/vercel/next.js/assets/106406594/d26e23c7-3278-4d89-8bab-d097ab287b42)
now if you answer both of the question as "NO" as below
![image](https://github.com/vercel/next.js/assets/106406594/ae4c4b2f-c71c-48db-a17b-d16788c03c80)

then you will find Pages folder like 
![image](https://github.com/vercel/next.js/assets/106406594/944556db-96b3-496c-8e41-a704a64eea17)
